utility/parseCSV.py

```diff
@@ -0,0 +1,83 @@
+# Requirements: 
+# 1. Entering data spreadsheets to the function in order as referenced.
+# 2. Updating priority field on both spreadsheets, ensuring value is equal on both. 
+
+import pandas as pd
+
+# parseCSV will accept the BOM.csv and MS_Quote Excel File and return two seperate Excel/CSV files with winning parts. 
+def parseCSV(Bom, MS_Quote): 
```

Comment:
See overall comment - rename this too please!

Reply:
+ also can we rename the variables to bom_file and MS_quote_file? more descriptive